src/cmd/run-smoketest/main.go

```diff
@@ -0,0 +1,196 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"crypto/md5"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/sago35/ochan"
+	"golang.org/x/sync/errgroup"
+)
+
+func main() {
+	threads := flag.Int("threads", 4, "threads of make smoketest")
```

Comment:
I think it may be reasonable to default this to `runtime.NumCPU()`?

Reply:
Thank you for reviewing it.
Changed.